src/components/breadcrumbs/Breadcrumbs.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import './Breadcrumbs.css'
+
+function makeBread (root) {
+  if (root.endsWith('/'))
+    root = root.substring(0, root.length - 1)
```

Comment:
We're using `standard` on this project, so this triggers a lint error. Can you update the .travis.yml to ensure that `npm run lint` is run on ci. Can your editor be configured to show lint errors as you type? I find that useful.

Also we now have a mix of unix and windows line endings. I'll add an editorconfig and a gitattributes file to the revamp branch to fix it.

- http://editorconfig.org/
- https://help.github.com/articles/dealing-with-line-endings/#platform-all